.gitignore

```diff
@@ -37,6 +37,7 @@ nx-cloud.env
 
 # End to End tests
 playwright-report/
+packages/plugin-e2e/test-results/
```

Comment:
All of a sudden Playwright started generating zip files related to reports. Anyway, this dir should definitely be ignored.